srcpkgs/ly/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'ly'
+pkgname=ly
+version=0.5.2
+revision=1
+_argoat_githash=0eb7afae7a001094c9166a8959e021375707522c
+_testoasterror_githash=ee7c9d031d4632a6f381a6c174a38539bac04068
+_configator_githash=8cec1786196ae6f6a8b35e66181277457f2a2bb2
+_ctypes_githash=eb4b36559de8d17015f9416861ed085d06f5f5ed
+_dragonfail_githash=0a2492c6aab3ff64e182db894ce4d40b26799fbd
+_termbox_githash=23fff64470b0730959b39c70aa31fbddd776d9bc
+build_style=gnu-makefile
+make_build_args="FLAGS=\"$CFLAGS\""
+conf_files="/etc/ly/config.ini"
+hostmakedepends="git"
+makedepends="ncurses-devel pam-devel xcb-util-devel libX11-devel"
+depends="pam util-linux xauth"
+short_desc="TUI (ncurses-like) display manager"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="WTFPL"
+homepage="https://github.com/nullgemm/ly"
+distfiles="https://github.com/nullgemm/ly/archive/v${version}.tar.gz
+ https://github.com/nullgemm/argoat/archive/${_argoat_githash}.tar.gz>argoat-${_argoat_githash}.tar.gz
+ https://github.com/nullgemm/testoasterror/archive/${_testoasterror_githash}.tar.gz>testoasterror-${_testoasterror_githash}.tar.gz
+ https://github.com/nullgemm/configator/archive/${_configator_githash}.tar.gz>configator-${_configator_githash}.tar.gz
+ https://raw.githubusercontent.com/nullgemm/ctypes/${_ctypes_githash}/ctypes.h>ctypes-${_ctypes_githash}.h
+ https://github.com/nullgemm/dragonfail/archive/${_dragonfail_githash}.tar.gz>dragonfail-${_dragonfail_githash}.tar.gz
+ https://github.com/nullgemm/termbox_next/archive/${_termbox_githash}.tar.gz>termbox_next-${_termbox_githash}.tar.gz"
+checksum="3460c2c8015add2946f9ff5d3e4bf051d7c07005698fe61279cedacd1283e010
+ 65b0b5d688f8538513f335795156441365b9cb319a28d2defbb29d81ea43a188
+ d627845bcece46499801bbef5b2ccf3e0bfe39493b52a16ae34478e54ab11bd6
+ 2067100f9779eceb5abefa9402ed56546314ca5ad6b3d5156d3e1eb46c853679
+ ef169974c68b548dc8ac83bd4714622775b024fab07dc2b7fe96e8e8f102dbab
+ d5cdd6d528a43c35d08e7fcac849370c07d949671ef592cfecb02b23ad3c4dad
+ 76d73183d2191e9e37285f39f2b1bd51f1dc2d991637902ff3c1aa35111b288d"
+skip_extraction="ctypes-${_ctypes_githash}.h"
+
+post_extract() {
+	cp -rv ../argoat-${_argoat_githash}/* sub/argoat/
+	cp -rv ../testoasterror-${_testoasterror_githash}/* sub/argoat/sub/testoasterror/
+	cp -rv ../configator-${_configator_githash}/* sub/configator/
+	cp -rv $XBPS_SRCDISTDIR/$pkgname-$version/ctypes-${_ctypes_githash}.h sub/ctypes/ctypes.h
+	cp -rv ../dragonfail-${_dragonfail_githash}/* sub/dragonfail/
+	cp -rv ../termbox_next-${_termbox_githash}/* sub/termbox_next/
+}
+
+_make() {
+	make CC="$CC" FLAGS="$CFLAGS"
+}
+
+pre_build() {
+	exit
```

Comment:
Why is this here?

Reply:
No idea, I inherited it from a previous version of the template. I'll remove it and see what happens.